packages/orbit-components/src/Loading/Loading.stories.tsx

```diff
@@ -1,123 +1,152 @@
+    controls: {
+      exclude: ["children", "customSize", "type"],
+    },
   },
-};
 
-InlineLoader.args = {
-  loadingText: "Please wait, content of the page is loading...",
+  args: {
+    customSize: undefined,
+    type: undefined,
+  },
 };
 
-export const CustomLoader = ({ customSize }) => {
-  return <Loading customSize={customSize} />;
+type CardLoadingStoryProps = React.ComponentProps<typeof Loading> & {
+  cardTitle: string;
+  cardDescription: string;
 };
 
-CustomLoader.args = {
-  customSize: 50,
-};
+export const CardLoading: StoryObj<CardLoadingStoryProps> = {
+  render: ({ loading, cardTitle, cardDescription, ...args }) => {
+    return (
+      <Card loading={loading} title={cardTitle} description={cardDescription}>
+        <Loading {...args} type="boxLoader">
+          <CardSection>
+            <Illustration name="EnjoyApp" size="medium" />
+          </CardSection>
+        </Loading>
+      </Card>
+    );
+  },
 
-export const Playground = ({ type, loadingText, loading, dataTest }) => {
-  return <Loading loading={loading} type={type} text={loadingText} dataTest={dataTest} />;
-};
+  args: {
+    cardTitle: "Card title",
+    cardDescription: "Card description",
+    text: undefined,
+    customSize: undefined,
+  },
 
-Playground.story = {
   parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    info: "Example of usage of Loading component inside Card. Check Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["children", "customSize", "text", "type"],
+    },
   },
 };
 
-Playground.args = {
-  type: TYPE_OPTIONS.PAGE_LOADER,
-  loadingText: "Please wait, content of the page is loading...",
-  loading: true,
-  dataTest: "test",
-};
+export const Playground: Story = {
+  render: ({ customSize, ...args }) => (
+    <div className="stroke-blue-dark">
+      <Loading {...args} customSize={customSize === 0 ? undefined : customSize} />
+    </div>
+  ),
+
+  args: {
+    loading: undefined,
+  },
 
-Playground.argTypes = {
-  type: {
-    options: Object.values(TYPE_OPTIONS),
-    control: {
-      type: "select",
+  parameters: {
+    info: "Playground of Loading component. Type buttonLoader inherits parent's stroke color. For visualisation of undefined value for customSize arg, set value number 0. Visit Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["children", "loading"],
     },
   },
 };
 
-export const Rtl = () => (
-  <RenderInRtl>
-    <Loading loading type="pageLoader" text="Please wait, content of the page is loading..." />
-  </RenderInRtl>
-);
+export const Rtl: Story = {
+  render: args => (
+    <RenderInRtl>
+      <Loading {...args} />
+    </RenderInRtl>
+  ),
 
-Rtl.story = {
-  name: "RTL",
+  args: {
+    loading: true,
+    type: TYPE_OPTIONS.INLINE_LOADER,
+    text: "Please wait, content of the page is loading...",
```

Comment:
Not sure how you will end up arranging the props based on my suggestions, but just pointing out that here are almost the same args as currently set on meta. But if you end up removing them from meta, then it's OK if they are not repeating the default values.

Reply:
now `text` will be included in `Default` story and I will re-use it in Playground using spreading.